test/helpers/Beer.js

```diff
@@ -21,7 +21,7 @@ class Beer {
 	}
 
 	createId() {
-		this.id = this.name;
+		this.id = this.name.toLowerCase();
```

Comment:
This would indicate that every user of this library would always have to make sure that the IDs they generate are lower case, or the system stops working. That doesn't seem very user friendly.